routes/documents.js

```diff
@@ -1,49 +1,49 @@
-const express = require("express");
+const express = require('express');
+
 const router = express.Router();
-const Document = require("../models/document");
-const { DocumentBodySchema } = require("./validators/validators");
-const validator = require("express-joi-validator");
+const Document = require('../models/document');
+const { DocumentBodySchema } = require('./validators/validators');
+const validator = require('express-joi-validator');
 
-router.get("/", (req, res, next) => {
+router.get('/', (req, res, next) => {
   Document.find({})
-    .populate("owner")
-    .populate("authors")
-    .then(found => {
+    .populate('owner')
+    .populate('authors')
+    .then((found) => {
       res.send(found);
     })
     .catch(next);
 });
 
-router.get("/:docId", (req, res, next) => {
+router.get('/:docId', (req, res, next) => {
   Document.findById(req.params.docId)
-    .populate("owner")
-    .populate("authors")
-    .then(record => {
-      console.log(record);
+    .populate('owner')
+    .populate('authors')
+    .then((record) => {
       res.send(record);
     })
     .catch(next);
 });
 
-router.post("/", validator(DocumentBodySchema), (req, res, next) => {
+router.post('/', validator(DocumentBodySchema), (req, res, next) => {
   Document.create(req.body)
-    .then(added => {
+    .then((added) => {
```

Comment:
I think more informative would be `document` instead of `added`. In other places too.

Reply:
Done